src/lib/shadow-shim-helper-rs/Cargo.toml

```diff
@@ -15,6 +15,7 @@ nix = { version = "0.27.1", features = ["event", "fs", "socket"] }
 log = { version = "0.4", features = ["release_max_level_debug"] }
 logger = { path = "../logger" }
 once_cell = "1.18.0"
+shadow-pod = { path = "../pod" }
```

Comment:
We've had some discussion about this before, but don't recall if we ever reached a consensus. I've gone back and forth and am probably responsible for both names in this case :sweat_smile: 

I've more recently been leaning towards hyphens since that seems to be the more established convention in rust crate names, but OTOH underscores don't have the "magical" transformation from hyphen to underscore as a rust identifier (e.g. package `shadow-pod` being imported as `use shadow_pod`)

I'm fine with picking one or the other and enshrining it in our [style guide](https://shadow.github.io/docs/guide/coding_style.html), but I think that'd make sense as a separate PR